lesson4/recursion13.js

```diff
@@ -0,0 +1,13 @@
+//compareArr([1,2,3],[1,2,3]) - true
+
+const compareArr = (a,b, i=0)=>{
+  if(i== a.length && i == b.length){
```

Comment:
`compareArr([1,3], [1,2,3,4])` should return false!

Reply:
it does return false. let me know if it doesn't work for other cases.